src/Components/test/testassets/ComponentsApp.App/ComponentsApp.App.csproj

```diff
@@ -7,7 +7,7 @@
 
   <ItemGroup>
     <Reference Include="Microsoft.Extensions.DependencyInjection" />
-    <Reference Include="Microsoft.AspNetCore.Components" />
+    <ProjectReference Include="..\..\..\Components\ref\Microsoft.AspNetCore.Components.csproj" />
```

Comment:
It's not much, but we can get the Component.App to start compiling against the ref assemblies. TestAssets is more valuable, but it's much more churn to do so since it references Blazor.